modules/oxide/src/less/theme/theme.less

```
@@ -53,6 +53,7 @@
 @import 'components/menubar/promotion';
 @import 'components/menubar/menubar-button';
 @import 'components/notification/notification';
+@import 'components/onboarding//onboarding';
```

I'm surprised that the double slash worked here.

```suggestion
@import 'components/onboarding/onboarding';
```